ui/test/nuxeo-selectivity.test.js

```diff
@@ -14,37 +14,114 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-import { fixture, html } from '@nuxeo/nuxeo-elements/test/test-helpers.js';
+
+import {
+  fixture,
+  flush,
+  html,
+  waitForAttrMutation,
+  waitForChildListMutation,
+} from '@nuxeo/nuxeo-elements/test/test-helpers.js';
+import { pressAndReleaseKeyOn } from '@polymer/iron-test-helpers/mock-interactions.js';
 import { dom } from '@polymer/polymer/lib/legacy/polymer.dom.js';
 import '../widgets/nuxeo-selectivity.js';
 
+/* eslint-disable no-unused-expressions */
 suite('<nuxeo-selectivity>', () => {
   let selectivityWidget;
   const data = ['Berlin', 'Lisbon', 'London', 'Rennes', 'Rome'];
-  setup(async () => {
-    selectivityWidget = await fixture(html`
-      <nuxeo-selectivity placeholder="No city selected" .data=${data} allow-clear></nuxeo-selectivity>
-    `);
+
+  suite('single value', () => {
+    setup(async () => {
+      selectivityWidget = await fixture(html`
+        <nuxeo-selectivity placeholder="No city selected" .data=${data}></nuxeo-selectivity>
```

Comment:
Just noticed that we are passing data attribute as `.data`. What is the reason why we use `dot`?

Reply:
That's how we pass on objects with lit-html.